hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/FsGetter.java

```diff
@@ -20,15 +20,15 @@
 import java.io.IOException;
 import java.net.URI;
 
-import org.apache.hadoop.classification.InterfaceAudience.Private;
+import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 
 /**
  * File system instance getter.
  */
-@Private
-class FsGetter {
+@InterfaceAudience.Public
```

Comment:
I'd be far happier with LimitedPrivate here. Certainly, you should also add an Evolving annotation also.

Reply:
changed it to InterfaceAudience.LimitedPrivate and added InterfaceStability.Evolving